pkg/restore/util.go

```diff
@@ -142,6 +142,19 @@ func GetSSTMetaFromFile(
 	}
 }
 
+// MakeDBPool makes a session pool with specficated size by sessionFactory.
+func MakeDBPool(size uint, sessionFactory func() (*DB, error)) ([]*DB, error) {
+	sessionPool := make([]*DB, 0, size)
+	for i := uint(0); i < size; i++ {
+		session, e := sessionFactory()
+		if e != nil {
+			return nil, e
```

Comment:
is it supposed to return the `dbPool` here, even on error? the warning in `restore.go:201` suggests so.